docs/firmware/3-zephyr-sdk/2-samples/dfu.md

```diff
@@ -17,7 +17,7 @@ Device Firmware Upgrade (DFU) procedure.
 Build Zephyr sample application for nRF9160 Feather:
 
 ``` {.console}
-$ west build -b circuitdojo_feather_nrf9160ns samples/dfu
+$ west build -b circuitdojo_feather_nrf9160ns samples/dfu -p -- -DCONFIG_MCUBOOT_IMAGE_VERSION=\"1.0.1\"
```

Comment:
This shoudn't be needed. If it is, then we need to update Zephyr SDK as well.

Additionally I prefer to use the same order of parameters (specifically writing about `-p` here) during each call to `west build`, because it is easier then to spot differences between each command.